machine/platform/register_darwin.go

```diff
@@ -0,0 +1,13 @@
+//go:build darwin
+// +build darwin
+
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2022, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+package platform
+
+func unixVariantStrategies() map[Platform]*Strategy {
+	// Nothing added for Darwin
+	return map[Platform]*Strategy{}
+}
```

Comment:
@nderjung If we get this PR in with stubs, I also want to get the FreeBSD PR in :D

Reply:
Yessir!